<ui version="4.0" >
 <class>RssRegexpDlg</class>
 <widget class="QDialog" name="RssRegexpDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>282</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Regular expression</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>382</width>
        <height>197</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>Main</string>
      </attribute>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QLabel" name="label_2" >
         <property name="text" >
          <string>RSS feed</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2" >
        <widget class="QComboBox" name="comboFeed" />
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>Expression</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2" >
        <widget class="QLineEdit" name="lineExpression" />
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="label_3" >
         <property name="text" >
          <string>Destination queue</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="2" >
        <widget class="QComboBox" name="comboQueue" />
       </item>
       <item row="3" column="0" >
        <widget class="QLabel" name="label_4" >
         <property name="text" >
          <string>Destination directory</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" >
        <widget class="QLineEdit" name="lineTarget" />
       </item>
       <item row="3" column="2" >
        <widget class="QToolButton" name="toolBrowse" >
         <property name="text" >
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="3" >
        <widget class="QGroupBox" name="groupBox" >
         <property name="title" >
          <string>Expression test</string>
         </property>
         <layout class="QGridLayout" >
          <item row="0" column="0" >
           <widget class="QLabel" name="label_5" >
            <property name="text" >
             <string>Input text</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QLabel" name="labelMatch" >
            <property name="minimumSize" >
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
            <property name="maximumSize" >
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
            <property name="scaledContents" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="2" >
           <widget class="QLineEdit" name="lineText" />
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>382</width>
        <height>197</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>TV show support</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout" >
       <item row="0" column="0" colspan="2" >
        <widget class="QRadioButton" name="radioTVSNone" >
         <property name="text" >
          <string>Not a TV serie</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2" colspan="2" >
        <widget class="QRadioButton" name="radioTVSSeason" >
         <property name="text" >
          <string>Season based TV serie</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2" >
        <widget class="QRadioButton" name="radioTVSEpisode" >
         <property name="text" >
          <string>Episode based TV serie</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2" colspan="2" >
        <widget class="QRadioButton" name="radioTVSDate" >
         <property name="text" >
          <string>Date based TV serie</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="label_6" >
         <property name="text" >
          <string>From</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" >
        <widget class="QLineEdit" name="lineTVSFrom" />
       </item>
       <item row="2" column="2" >
        <widget class="QLabel" name="label_7" >
         <property name="text" >
          <string>To</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3" >
        <widget class="QLineEdit" name="lineTVSTo" />
       </item>
       <item row="3" column="0" colspan="2" >
        <widget class="QCheckBox" name="checkTVSRepacks" >
         <property name="text" >
          <string>Download repacks</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2" colspan="2" >
        <widget class="QCheckBox" name="checkTVSTrailers" >
         <property name="text" >
          <string>Download trailers/teasers</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="4" >
        <widget class="QCheckBox" name="checkTVSNoManuals" >
         <property name="text" >
          <string>Exclude episodes downloaded manually</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="4" >
        <widget class="QLabel" name="labelManage" >
         <property name="text" >
          <string>&lt;a href="manageDownloaded">Manage the list of  downoaded episodes</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>272</width>
       <height>41</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RssRegexpDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>227</x>
     <y>237</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>258</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RssRegexpDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>295</x>
     <y>243</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>258</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
